recipes/loompy/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "loompy" %}
+{% set version = "1.0.2" %}
+{% set sha256 = "36159c7d1423d4beb9848efc17ca5b2893398f1c7a6dccb408a413a112134c2b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  skip: True  # [win]
```

Comment:
I don't see any indication that windows is not supported. Is there a reason to skip windows?

Reply:
I am one of the authors of the library. `loompy` was never tested on windows, I just guessed the policy of the community is to have only packages that work and not packages that maybe work. It might still work for windows, but a comprehensive test is beyond my bandwidth right now. I can remove the preprocessing comment if appropriate.

Reply:
sounds reasonable to leave windows out then.